CustomeDB/LinkedList.cpp

```diff
@@ -120,6 +131,21 @@ void LinkedList::addAtEnd(string name, int id, string phone, string address, int
 
 }
 
+void LinkedList :: addAtEnd(string name, int id, string description, int client_id, int money_earned, string deadline, string complete_date) {
+    Node* temp = new Node(name, id, description, client_id, money_earned, deadline, complete_date);
+    if (prjhead == NULL) {
+        prjhead = temp;
+    }
+    else {
+        prjtail = prjhead;
+        while (prjtail->next != NULL) {
+            prjtail = prjtail->next;
+        }
+        prjtail->next = temp;
+        prjtail = temp;
```

Comment:
why are both the iterator(prjtail) and its next pointing to same object? and why isn't temp's next NULL ?